<launch>
	<param name = "convex_hull/reference_frame" type = "string" value = "/world" />
	<param name = "convex_hull/cloud_input_topic" type = "string" value = "/selected_points" />
	<param name = "convex_hull/openrave_show_grasps" type = "bool" value = "true" />
	<param name = "convex_hull/using_atlas" type = "bool" value = "false" />
	<param name = "convex_hull/debug_and_logging" type = "string" value = "full" />
	<param name = "convex_hull/perception_link" type = "string" value = "/camera_link" />

	<param name = "convex_hull/output_pose_frame" type = "string" value = "/pelvis" />
	<param name = "convex_hull/mesh_ref_frame" type = "string" value = "/pelvis" />
</launch>
